README.md

```diff
@@ -1 +1,8 @@
-# myre2
\ No newline at end of file
+# myre2
+
+
+def myfunc(lis):
+    lis.sort()
+    return lis[-1]
+print(myfunc([1,11,7,8,-1,6]))
```

Comment:
You must follow my example pattern. It will help by running script will have results too.